tests/e2e/runner/__tests__/minter-suite-v2/i-filtered-shared-da-exp-settlement.test.ts

```diff
@@ -22,7 +22,8 @@ import { Logger } from "@ethersproject/logger";
 Logger.setLogLevel(Logger.levels.ERROR);
 
 // waiting for subgraph to sync can take longer than the default 5s timeout
-jest.setTimeout(30 * 1000);
+// @dev specifically for this test, we need to wait for auction to complete, so need extra time
+jest.setTimeout(75 * 1000);
```

Comment:
why 75? would like some context/clarity on the hardcoded number added to the comment

Reply:
good call - updated with some additional comments + reduced to 60s in [0458c18](https://github.com/ArtBlocks/artblocks-subgraph/pull/247/commits/0458c18608e0b7218d25fdf13206fc680632ef5c)

for additional context here, our e2e tests really do test things end-to-end, so in this case, we need to really run an auction, which is configured as 22.5 seconds. If we try and get too precise and shorten more, we would risk missing the target intermediate point in the auction. Time is a bit precise because we have a jest hardhat script talking to the locally running blockchain, and locally running subgraph, which is also talking to the local blockchain.